Packs/illuminate/Integrations/Analyst1/Analyst1.yml

```
@@ -598,7 +605,155 @@ script:
       description: The AlienVault OTX vendor.
       type: String
     description: Queries the Analyst1 REST API and enriches the given URL with Analyst1 indicator data.
+  - name: analyst1-get-sensor-taskings
+    arguments:
+    - name: sensor_id
+      required: true
+      description: Sensor ID number for this Analyst1 instance.
+    - name: timeout
+      default: true
+      description: Overrides the XSOAR default of 10s for timeout. Default Analsyt1 app is 200s for this command. Caller may further override as required.
+      defaultValue: "200"
+    outputs:
+    - contextPath: Analyst1.SensorTaskings
+      description: 'Root JSON for processing all taskings data. '
+    - contextPath: Analyst1.SensorTaskings.id
+      description: Sensor ID for this Taskings response.
+    - contextPath: Analyst1.SensorTaskings.version
+      description: Current version of the Sensor.
+    - contextPath: Analyst1.SensorTaskings.Indicators
+      description: Current array of Indicators tasked
+    - contextPath: Analyst1.SensorTaskings.Rules
+      description: Current array of Signatures tasked
+    description: 'Queries the Analyst1 REST API for the current sensor taskings given a valid Sensor ID. This can be used to start subscription to an Sensor ID. The result gives the version (which can later be used to invoke ''diff'') and all current taskings. Note: This operation may trigger XSOAR to "oversize" the task built on this automation. If so, you may need to turn off quiet mode explicitly. The analyst1-get-sensor-config can alternatively be used to get a simple text file of current indicators or signatures.'
+  - name: analyst1-get-sensors
+    arguments:
+    - name: page
+      description: page of Sensors to iterate
+      defaultValue: "1"
+      type: unknown
+    - name: pageSize
+      default: true
+      description: size of each page of Sensors to iterate. Maximum 50.
+      defaultValue: "50"
+    description: Queries the Analyst1 REST API to retrieve a list of registered sensors.
+  - name: analyst1-batch-check
+    arguments:
+    - name: values
+      required: true
```

```suggestion
      required: true
      isArray: true
```
See [here](https://xsoar.pan.dev/docs/integrations/yaml-file#command-arguments).